src/Libraries/RevitNodes/Properties/Resources.resx

```diff
@@ -369,4 +369,7 @@
   <data name="ScheduleTypeArgumentException" xml:space="preserve">
     <value>Invalid Schedule Type.</value>
   </data>
+  <data name="ScheduleExportError" xml:space="preserve">
```

Comment:
en-us, and this resx file don't seem to be the same, is this one missing the  `ScheduleExportError`?